app/src/main/java/com/dessalines/thumbkey/ui/components/keyboard/KeyboardKey.kt

```diff
@@ -363,7 +364,15 @@ fun KeyText(key: KeyC, keySize: Dp, hideLetters: Boolean, capsLock: Boolean) {
         }
         is KeyDisplay.TextDisplay -> {
             // Only  hide the letters for text, not symbols
-            if (!hideLetters) {
+            val containsLetters = display.text.any { it.isLetter() }
```

Comment:
I fixed this to be clearer, and not use a triple negative

Reply:
This actually changes the logic, but after thinking it over, I do agree that this is probably the better of the two choices here anyway.